.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,20 @@
+name: Style Check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  clang-format:
+    name: Clang-Format
```

Comment:
```suggestion
    name: clang-format
```
to match the name of the actual command being run?

Reply:
the workflow name `name: Style Check` and the job name `name: Clang-Format` in the end are displayed in the list of checks like this.
![image](https://user-images.githubusercontent.com/8142186/119234238-1c70a980-bb2d-11eb-9fc8-cb6bed40b69a.png)
the style is called "Clang-Format" (https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormatStyleOptions.html#), the toolset "ClangFormat" https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormat.html and the command "clang-format".
I don't mind changing it, but if the name (line 11) is the same as in line 10 you could actually remove it

Reply:
> the workflow name `name: Style Check` and the job name `name: Clang-Format` in the end are displayed in the list of checks like this.

Yep, I'm aware of it and that's why suggested modifying it slightly. I think "Coding Style Check" may be a bit clearer, especially as there is work on Language Style Guides in the project as well?

> the style is called "Clang-Format" (https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormatStyleOptions.html#), the toolset "ClangFormat" https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormat.html and the command "clang-format".

Ok. I'm not fussed about it, we can also keep it like that.

Maybe someone else can comment as well.

Reply:
Based on their Language Style Guide, we'd be after "Coding Style Check / Verify Clang-Format compliance" for the check title, so:
- workflow ` .github/workflows/coding-style-checks.yml`, `name: Coding Style Check`
- `jobs` `verify-clang-format`, `name: Verify Clang-Format compliance`

Reply:
>     * workflow ` .github/workflows/coding-style-checks.yml`, `name: Coding Style Check`
>     * `jobs` `verify-clang-format`, `name: Verify Clang-Format compliance`
I have changed the names accordingly